### PR TITLE
Added article "Clear Browsing Data"

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/clear-browsing-data?branch=pr-en-us-1737

To traverse API links: in a 404 page, double-click "review" in the Address bar > Delete > Enter.

### CODE REVIEW

File: microsoft-edge/webview2/concepts/clear-browsing-data.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+Clear Browsing Data APIs used in this example:
+
+* [ClearBrowsingDataInTimeRange(dataKinds, startTime, endTime)](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprofile4#clearbrowsingdataintimerange)
+* [COREWEBVIEW2_BROWSING_DATA_KINDS](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcompositioncontroller4#corewebview2_browsing_data_kinds)
+
+Other APIs:
+
+* [ICoreWebView2ClearBrowsingDataCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalclearbrowsingdatacompletedhandler).
+* [ICoreWebView2Profile](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalprofile2).
+* `get_Profile` <!-- not found-->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/clear-browsing-data.md:164]: @jasonstephen15 Can you get a URL to link to?

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/clear-browsing-data.md:144]: > Did we decide if we wanted to only include APIs that are specific to the Clear Browsing Data API?

Yes, below a sample listing, we only want to list the APIs that belong to the API which is being doc'd.

Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/clear-browsing-data.md:144]: @jasonstephen15 - are there any APIs outside of the Clear Browsing Data APIs that we need to doc, such as link to?

@Reezaali

**Review by maurawinstanley — APPROVED** [review]
Approving from dev perspective of Clear Browsing Data API, I would recommend waiting for Jason's review before merging.

File: microsoft-edge/webview2/concepts/clear-browsing-data.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 02/09/2022
+---
+# Clear browsing data from the user data folder
+
+To clear browsing data from the user data folder for a WebView2 app to free up space, call the methods of the Clear Browsing Data API.
+
+The Clear Browsing Data API allows you to programmatically clear space in the user data folder for a user profile.  These methods clear selected browsing data that is stored in the user data folder of a WebView2 app.  For example, you can delete browsing data when a user signs out, to clear space in the user data folder.
```
Comment (jasonstephen15) [inline microsoft-edge/webview2/concepts/clear-browsing-data.md:15]:
```suggestion
The Clear Browsing Data API allows you to programmatically erase data in user data folder associated with a webview2 user profile. An example use case of this is if an app wants to clear user data/history when a user signs out. 
```

**Review by jasonstephen15 — APPROVED** [review]

**Pull request merged**